### DISCUSSION TITLE
Spectrum analyzer issue on 0.21 release

### DISCUSSION THREAD

**Discussion by g2pa** [egzumer/uv-k5-firmware-custom#308] [General]
I have only just started using the egzumer release 0.21 on a UV-K6 and a K5, at first everything seemed great, I'm now trying the Spectrum analyzer and have found a minor issue, when I set the Bandwidth to 25k and the sweep to 25k it's working fine, if I try and set the sweep to 50k (to get more bandwidth), after it receives a signal, the analyzer seems to get stuck on that frequency for a long time. It does eventually release it but the whole spectrum scope freezes until that single frequency is released with just static. This is even after the received signal has really gone. Setting the sweep back to 25k does seem to fix it. Am I doing something wrong or could this a new issue with 0.21 and the new sweep offerings?

**Comment by iu0azb** [comment]
I've noted too along with other issue. Not always but diving the scope with A&B buttons, if a band edge is crossed, later going back to the previous band is not more really received anymore.. this could be reproduced by using a stable transmission next to a band edge..in my case I use the unwanted harmonic at 130mhz that I've tried to mitigate with additional shielding ;) 

I would suggest to study the option to lockout from reception activity any of these harmonics since annoying stop scanning ops and affect auto tune on band scope

**Reply by iu0azb** [reply]
Let me add that I noted this bug only on 0.21

**Comment by gartnl** [comment]
I've noticed simular even with 25/25 settings. After a signal is received that signal never drops below the squelch level anymore. It shows like it is still receiving something but that is only static. Looking at the setting with PTT and exit releases it. 
Reverted back to 20.1 and there it is normal again.

**Comment by pietss** [comment]
Perhaps what is being said here has a bearing on what I said here a few days ago https://github.com/egzumer/uv-k5-firmware-custom/discussions/305 
Compared to what I had written, I add one thing, so perhaps I can make it better understood, in extremely simple words given that I am not an expert and I don't know English.
In spectrum mode what I was saying in the other post also happens in a second slightly different way, that is when continuous signals are detected (the nature of which I don't know but it is not verbal communications) and the line is raised of the squelch, this involves the increase in the detected intensity of the signal in question because the bar is "hooked" by the squelch line and therefore raised up to where the squelch line rises.
